examples/README.md

```diff
@@ -72,6 +72,10 @@
 
 [vmf_clothing.py](vmf_clothing.py) - Visual Matrix Factorization (VMF) with Amazon Clothing dataset.
 
+### Sentiment
+[hypar_example.py](hypar_example.py) - Hypergraphs with Attention on Reviews (HypAR) with Cellphone dataset.
```

Comment:
Please update this example link as well.

Reply:
Removed in commit d0cfe0b